code/modules/antagonists/pirate/pirate_gangs.dm

```diff
@@ -115,3 +115,37 @@ GLOBAL_LIST_INIT(heavy_pirate_gangs, init_pirate_gangs(is_heavy = TRUE))
 	response_received = "Thanks for the credits, landlubbers."
 	response_too_late = "Too late to beg for mercy!"
 	response_not_enough = "Trying to cheat us? You'll regret this!"
+
+///Expirienced formed employes of Interdyne Pharmaceutics now in a path of thievery and reckoning
+/datum/pirate_gang/grey
+	name = "Expharmacist Unrest"
+
+	is_heavy_threat = TRUE
+	ship_template_id = "interdyne"
+	ship_name_pool = "interdyne_names" //just points to THE ONE AND ONLY
+
+	threat_title = "Fundings for Research"
+	threat_content = "Greetings, this is the %SHIPNAME. We require some funding for our operations. \
+		%PAYOFF credits should suffice."
+	arrival_announcement = "We humbly ask for a substantial amount of income for the future research of our cause. It sure would be a shame if you got sick, but we can fix that."
+	possible_answers = list("Very well.","Get a job.")
+
+	response_received = "Thanks for your generosity, your money wont be wasted."
+	response_too_late = "We hope you like skin cancer!"
+	response_not_enough = "This is not nearly enough for our operations. I'm afraid we'll have to borrow some."
+
+///Previous Nanotrasen Assitant workers fired for many reasons now looking for revenge and your bank account.
+	name = "The Grey Tide-6"
+
```

Comment:
You didn't document this pirate faction at all in the PR bro

Reply:
Also, the Grey Tide-6? Come on man that's awful, just make it the grey tide

Reply:
sure I dont really care